### DISCUSSION TITLE
请问制作双语epub电子书功能，是调用chatGPT对电子书的内容进行翻译吗

### DISCUSSION THREAD

**Discussion by llj1998** [immersive-translate/immersive-translate#1157] [Help]
### 概述 | Summary

已开通会员，请问制作双语epub电子书功能，是调用chatGPT对电子书的内容进行翻译吗
>
> 设置里的翻译服务已经选了openAI，服务提供商选的是PRO会员

**Comment by theowenyoung** [comment]
是的。用的GPT3.5
